help/articles/101013/index.html

```diff
@@ -121,15 +121,15 @@ <h3>Reporting Private Messages (PM's)</h3>
         <h3>On the Desktop Clients:</h3>
         <ol>
             <li>Right-click the message from the chat you wish to report. Alternatively, you can left-click and hold, simulating a long-press on mobile.</li>
-            <li>Press on the "Report Message" option from the Dropdown Menu.</li>
+            <li>Click on the "Report Message" option from the Dropdown Menu.</li>
             <li>Follow along the walkthrough, <b>making sure you read it <span style="text-decoration: underline;">carefully</span></b>.</li>
             <li>You are all set. Please note that as stated in the walkthrough, the content of the message you reported will be sent, <span style="text-decoration: underline;">unencrypted</span>, to our Review Team, along with the ID of the message, message chat and the user who <span style="text-decoration: underline;">sent</span> the message.</li>
         </ol>
         <br>
         <h3>On the Mobile Clients:</h3>
         <ol>
             <li>Long press on the message you wish to report.</li>
-            <li>Press on the "Report Message" option from the Dropdown Menu.</li>
+            <li>Click on the "Report Message" option from the Dropdown Menu.</li>
```

Comment:
As @3ux4 mentioned, you can't (normally) click on a button on a mobile client. That's why we worded it as "press," which is understood as clicking where touching is not an option.

Feel free to leave the "click" correction on PC (bear in mind touchscreen laptops and people who use touchpads), but the mobile client's "click" is redundant and (under normal circumstances, considering most people don't use their phone/tablet as a PC or don't have damaged touchscreens which require them to use a mouse or other external entry device)